library/alloc/src/collections/btree/navigate.rs

```diff
@@ -42,50 +42,46 @@ where
 
     let mut min_node = root1;
     let mut max_node = root2;
-    let mut min_found = false;
-    let mut max_found = false;
+    let mut min_bound = Some(start);
+    let mut max_bound = Some(end);
```

Comment:
Can we place a comment on what the meaning of Some/None here is?

Reply:
How about a commented type?

@rustbot modify labels: -S-waiting-on-author +S-waiting-on-review